packages/synapse-react-client/src/mocks/mockCertificationQuiz.ts

```diff
@@ -1,67 +1,112 @@
+import {
+  MULTICHOICE_RESPONSE_CONCRETE_TYPE_VALUE,
+  MultichoiceResponse,
+} from '@sage-bionetworks/synapse-types'
 import { ObjectType } from '@sage-bionetworks/synapse-types'
 import { Quiz, PassingRecord } from '@sage-bionetworks/synapse-types'
 
+const question1 = {
```

Comment:
Reworked mock certification quiz objects.  Reuse questions, and provide a function for creation of an answer (MultichoiceResponse)